web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-fifty-text-media.php

```diff
@@ -0,0 +1,37 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-oobrzu.php, but found phpcs-scan-oObRZu.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).